PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Summary
+
+## Problem description
+
+## Pros/cons of approach implemented
+<!-- Expectations for PRs: https://safetyculture.atlassian.net/wiki/spaces/ENG/pages/2881716914/RFC66+Pull+Request+Code+Review+Standards -->
+
+<!-- Have you included a short summary of what the PR does, a description of the problem being solved and the pros/cons of the approach? -->
```

Comment:
Maybe move these to the top of the file, as overall guidance?

Reply:
Moved to the top in latest change